api/query/atoms_test.go

```diff
@@ -368,7 +383,56 @@ func TestStructuredQuery_bindStatusSomeRuns(t *testing.T) {
 			},
 		},
 	}
-	assert.Equal(t, expected, q.BindToRuns(runs))
+	assert.Equal(t, expected, q.BindToRuns(runs...))
+}
+
+func TestStructuredQuery_bindExists(t *testing.T) {
```

Comment:
I know it makes the test longer, but could include a test that's more realistic? Multiple branches to the exists and multiple runs (with different browser names). That scenario is what the exists structure was created for.

Reply:
I also agree that we should have multiple branches in the top level. The most recent change added multiple runs, but there's still only one `And` in the top-level `Exists`.

Reply:
Done (added a separate BrowserStatusNeq).